robottelo/cli/factory.py

```diff
@@ -1735,7 +1735,7 @@ def setup_org_for_a_custom_repo(options=None):
     }
 
 
-def _setup_org_for_a_rh_repo(target_sat, options=None):
+def _setup_org_for_a_rh_repo(options=None):
```

Comment:
Remove this function if it is not being used anywhere, or move under `cli_factory` class or appropriate mixins, so that no one uses it from this module again.

The ultimate goal is get rid of this entire `cli/factory.py` module and use `cli_factory` class.

Reply:
All the tests should `setup_org_for_a_rh_repo` from `cli_factory.py`

Reply:
I understand the goal of the helpers refactor and the need to standardize on `robottelo/host_helpers/cli_factory`, but there is enough going on in this PR already that I think this change should be part of a separate effort, as you suggest above. What do you think?

Reply:
Ok, Cool Lets move on and track that in separate PR, I hope someone from our earlier effort would take it up ? 

Ping @shweta83 / @ColeHiggins2  / @sambible  ??